PhysicsTools/PatAlgos/plugins/PATMuonProducer.cc

```diff
@@ -980,68 +970,23 @@ void PATMuonProducer::produce(edm::Event& iEvent, const edm::EventSetup& iSetup)
       muon.setSelector(reco::Muon::PuppiIsoTight, puppiCombinedIsolationPAT < 0.12);
     }
 
-    float jetPtRatio = 0.0;
-    float jetPtRel = 0.0;
-    float mva = 0.0;
-    float mva_lowpt = 0.0;
-    if (computeMuonMVA_ && primaryVertexIsValid && computeMiniIso_) {
-      if (mvaUseJec_) {
-        mva = globalCache()->muonMvaEstimator().computeMva(muon,
-                                                           primaryVertex,
-                                                           *(mvaBTagCollectionTag.product()),
-                                                           jetPtRatio,
-                                                           jetPtRel,
-                                                           miniIsoValue,
-                                                           mvaL1Corrector.product(),
-                                                           mvaL1L2L3ResCorrector.product());
-        mva_lowpt = globalCache()->muonLowPtMvaEstimator().computeMva(muon,
-                                                                      primaryVertex,
-                                                                      *(mvaBTagCollectionTag.product()),
-                                                                      jetPtRatio,
-                                                                      jetPtRel,
-                                                                      miniIsoValue,
-                                                                      mvaL1Corrector.product(),
-                                                                      mvaL1L2L3ResCorrector.product());
-
+    std::vector<float> jetPtRatioRel = {0.0, 0.0};
```

Comment:
a `std::array<float, 2>` (or even a fixed size C array) is certainly more efficient here